app/templates/courses/teacher-courses-view.jade

```diff
@@ -93,3 +94,7 @@ mixin course-info(course)
           | (
           span(data-i18n='teacher.guides_coming_soon')
           | )
+    if campaign && campaign.get('levelsUpdated')
+      p.small.m-t-2
+        span.spr(data-i18n="courses.last_updated")
```

Comment:
I've been liking a one-line `| ` instead of `.spr` so that the space is "real" instead of a CSS space that doesn't get selected/etc when you highlight/copy. Not super important here though.